test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/ClassValueTest.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.lang
+
+import org.junit.Test
+import org.junit.Assert._
+
+class ClassValueTest {
+  @Test def testClassValue(): Unit = {
+    val classValue = new ClassValue[Any] {
+      private var counter: Int = 0
+
+      // () and null are corner-cases of our implementation
```

Comment:
Just realized that this is not true anymore.

Reply:
Hum, yes, it's still true, since we need the addition `has`/`containsKey` tests for them, and they are explicitly mentioned in the code.